tests/readyflag/impl/ReadyFlagCppImplementation.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
```

Comment:
License header is missing

Reply:
License header is not any of the CPP Implementation Files in Auto Tests .So Shall I add here ?

Reply:
hmm, I don't see were the license header would be missing, could you point me to it (file path)?

Reply:
CombinedTestsCppImplementation.h for example